tests/travis_tests.fish

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env fish
+
+set -l return_code 0
+
+omf help
+omf query PATH
+omf install apt
+
+exit $return_code
```

Comment:
Because it was a draft, I didn't really pay attention to this.

I'll push another one with a small draft of what we may want here.